packages/eslint-plugin/docs/rules/no-wrapper-object-types.mdx

```diff
@@ -10,13 +10,12 @@ import TabItem from '@theme/TabItem';
 > See **https://typescript-eslint.io/rules/no-wrapper-object-types** for documentation.
 
 The JavaScript language has a set of language types, but some of them correspond to two TypeScript types, which look similar: `boolean`/`Boolean`, `number`/`Number`, `string`/`String`, `bigint`/`BigInt`, `symbol`/`Symbol`, `object`/`Object`.
-The difference is that the lowercase variants are compiler intrinsics and specify the actual _runtime types_ (that is, the return value when you use the `typeof` operator), while the uppercase variants are _structural types_ defined in the library that can be satisfied by any user-defined object with the right properties, not just the real primitives.
+The difference is that the lowercase variants are compiler intrinsics and specify the actual _runtime types_ (that is, the type indicated when executing `typeof x` at runtime), while the uppercase variants are _structural types_ defined in the library that can be satisfied by any user-defined object with the right properties, including but not limited to the real primitives.
```

Comment:
made slight edit here to try to clarify that it's about the _runtime_ `typeof` operator in TS vs the type-time `typeof` operator, which also exists.